### PR TITLE
dev, v3.0, v3.1, v2.1: update lightning download link and importer sample config

### PR DESCRIPTION
**Pull request by amyangfei**
### What is changed, added or deleted? 

- Fix tidb-lightning download link
- Fix tikv-importer advertise-addr in sample config

### Which version does your change affect? 

dev/v3.0/v3.1/v2.1

### CODE REVIEW

**Comment by amyangfei** [comment]
PTAL @kennytm @yikeke

File: dev/reference/tools/tidb-lightning/deployment.md [hunk 1]
```diff
@@ -181,9 +181,9 @@ TiDB Lightning 可随 TiDB 集群一起用 [Ansible 部署](/dev/how-to/deploy/o
 
 通过以下链接获取 TiDB Lightning 安装包（需选择与集群相同的版本）：
 
-- **v2.1.9**: [https://download.pingcap.org/tidb-v2.1.9-linux-amd64.tar.gz](https://download.pingcap.org/tidb-v2.1.9-linux-amd64.tar.gz)
-- **v2.0.9**: [https://download.pingcap.org/tidb-lightning-v2.0.9-linux-amd64.tar.gz](https://download.pingcap.org/tidb-lightning-v2.0.9-linux-amd64.tar.gz)
-- 最新 unstable 版本：[https://download.pingcap.org/tidb-lightning-test-xx-latest-linux-amd64.tar.gz](https://download.pingcap.org/tidb-lightning-test-xx-latest-linux-amd64.tar.gz)
+- [v2.1](https://pingcap.com/docs-cn/v2.1/reference/tools/download/#tidb-lightning)
+- [v3.0](https://pingcap.com/docs-cn/v3.0/reference/tools/download/#tidb-lightning)
+- [最新 unstable 版本](/dev/reference/tools/download.md#tidb-lightning)
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/deployment.md]:
```suggestion
- [最新开发版本 (unstable)](/dev/reference/tools/download.md#tidb-lightning)
```

File: v2.1/reference/tools/tidb-lightning/deployment.md [hunk 2]
```diff
@@ -171,9 +171,8 @@ TiDB Lightning 可随 TiDB 集群一起用 [Ansible 部署](/v2.1/how-to/deploy/
 
 通过以下链接获取 TiDB Lightning 安装包（需选择与集群相同的版本）：
 
-- **v2.1.9**：`https://download.pingcap.org/tidb-v2.1.9-linux-amd64.tar.gz`
-- **v2.0.9**：`https://download.pingcap.org/tidb-lightning-v2.0.9-linux-amd64.tar.gz`
-- 最新 unstable 版本：`https://download.pingcap.org/tidb-lightning-test-xx-latest-linux-amd64.tar.gz`
+- [v2.1](/v2.1/reference/tools/download.md#tidb-lightning)
+- [最新 unstable 版本](https://pingcap.com/docs-cn/dev/reference/tools/download/#tidb-lightning)
```
Comment (yikeke) [inline v2.1/reference/tools/tidb-lightning/deployment.md]:
```suggestion
- [最新开发版本 (unstable)](https://pingcap.com/docs-cn/dev/reference/tools/download/#tidb-lightning)
```

File: v3.0/reference/tools/tidb-lightning/deployment.md [hunk 3]
```diff
@@ -182,9 +182,9 @@ TiDB Lightning 可随 TiDB 集群一起用 [Ansible 部署](/v3.0/how-to/deploy/
 
 通过以下链接获取 TiDB Lightning 安装包（需选择与集群相同的版本）：
 
-- **v2.1.9**: `https://download.pingcap.org/tidb-v2.1.9-linux-amd64.tar.gz`
-- **v2.0.9**: `https://download.pingcap.org/tidb-lightning-v2.0.9-linux-amd64.tar.gz`
-- 最新 unstable 版本：`https://download.pingcap.org/tidb-lightning-test-xx-latest-linux-amd64.tar.gz`
+- [v2.1](https://pingcap.com/docs-cn/v2.1/reference/tools/download/#tidb-lightning)
+- [v3.0](/v3.0/reference/tools/download.md#tidb-lightning)
+- [最新 unstable 版本](https://pingcap.com/docs-cn/dev/reference/tools/download/#tidb-lightning)
```
Comment (yikeke) [inline v3.0/reference/tools/tidb-lightning/deployment.md]:
```suggestion
- [最新开发版本 (unstable)](https://pingcap.com/docs-cn/dev/reference/tools/download/#tidb-lightning)
```

File: v3.1/reference/tools/tidb-lightning/deployment.md [hunk 4]
```diff
@@ -181,9 +181,10 @@ TiDB Lightning 可随 TiDB 集群一起用 [Ansible 部署](/v3.1/how-to/deploy/
 
 通过以下链接获取 TiDB Lightning 安装包（需选择与集群相同的版本）：
 
-- **v2.1.9**: `https://download.pingcap.org/tidb-v2.1.9-linux-amd64.tar.gz`
-- **v2.0.9**: `https://download.pingcap.org/tidb-lightning-v2.0.9-linux-amd64.tar.gz`
-- 最新 unstable 版本：`https://download.pingcap.org/tidb-lightning-test-xx-latest-linux-amd64.tar.gz`
+- [v2.1](https://pingcap.com/docs-cn/v2.1/reference/tools/download/#tidb-lightning)
+- [v3.0](https://pingcap.com/docs-cn/v3.0/reference/tools/download/#tidb-lightning)
+- [v3.1](/v3.1/reference/tools/download.md#tidb-lightning)
+- [最新 unstable 版本](https://pingcap.com/docs-cn/dev/reference/tools/download/#tidb-lightning)
```
Comment (yikeke) [inline v3.1/reference/tools/tidb-lightning/deployment.md]:
```suggestion
- [最新开发版本 (unstable)](https://pingcap.com/docs-cn/dev/reference/tools/download/#tidb-lightning)
```

File: dev/how-to/get-started/tidb-lightning.md [hunk 5]
```diff
@@ -71,7 +71,7 @@ TiDB Lightning 主要包含两个部分:
 
     [server]
     # tikv-importer 监听的地址，tidb-lightning 需要连到这个地址进行数据写入。
-    addr = "0.0.0.0:8287"
+    addr = "127.0.0.1:8287"
```
Comment (kennytm) [inline dev/how-to/get-started/tidb-lightning.md]:
```suggestion
    addr = "192.168.20.10:8287"
```

use the machine's IP here, not 127.0.0.1.

**Comment by amyangfei** [comment]
PTAL @kennytm @yikeke

**Pull request merged**